xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties":{
+    "facility": {
+      "type": "string",
+      "description": "Facility Name (NSLS-II, APS, etc)"
+    },
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength in Angstrom, Depreciation Warning: this will depreciated in upcoming releases, please do not use this going forward"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
+    },
+    "bt_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Beamtime object"
+    },
+    "sample_name":{
+      "type": "string",
+      "description": "name of sample being measured"
+    },
+    "sample_composition":{
+      "type": "object",
+      "description": "dictionary of sample stoichiometry"
+    },
+    "sample_phase":{
+      "type": "object",
+      "description": "dictionary of sample phase information"
+    },
+    "sample_maker":{
+      "type": "array",
+      "description": "list of sample maker names "
+    },
+    "sa_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Sample object"
+    },
+    "cif_name":{
+      "type": "array",
+      "description": "list of possible structure filenames"
+    },
+    "collaborators":{
+      "type": "array",
+      "description": "list of collaborator names"
```

Comment:
add first last etc. if relevant (see comment above).  add "These may be people involved in the research but who didn't participate in the experiment, but you still want to link them in the metadata.  These are like tags that yield more powerful searches if used correctly"